runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/BatchModeExecutionContext.java

```diff
@@ -66,6 +66,8 @@
   // TODO: Move throttle time Metric to a dedicated namespace.
   protected static final String DATASTORE_THROTTLE_TIME_NAMESPACE =
       "org.apache.beam.sdk.io.gcp.datastore.DatastoreV1$DatastoreWriterFn";
+  protected static final String HTTP_CLIENT_API_THROTTLE_TIME_NAMESPACE =
```

Comment:
This seems an odd explicit namespace to use, is it for backwards compatibility reasons?

Reply:
Yes, it just followed the same convention for compatibility. Moving to a dedicated namespace is on TODO list.